src/Components/Search/Search.css

```diff
@@ -0,0 +1,29 @@
+.search--container {
+  position: relative;
+  display: flex;
+  align-items: center;
+}
+
+.search--input {
+  width: 100%;
```

Comment:
Is it intentional to have the search bar extend the whole width of the screen? It looks really odd on my large screens. We could discuss this.

Reply:
Yes, at least for mobile first design.  We could discuss what it would look like on larger screens.

Reply:
Definitely would be great discussion to address as we start implementing Responsive dimensions to each state and keeping that uniform throughout the app!